src/Generating/FinishGen.cpp

```diff
@@ -1395,3 +1395,369 @@ eMonsterType cFinishGenPassiveMobs::GetRandomMob(cChunkDesc & a_ChunkDesc)
 
 
 
+
+////////////////////////////////////////////////////////////////////////////////
+// cFinishGenOres:
+
+void cFinishGenOres::GenFinish(cChunkDesc & a_ChunkDesc)
+{
+	int seq = 1;
+
+	// Generate the ores from the ore list.
+	for (const auto & ore: m_OreInfos)
+	{
+		GenerateOre(
+			a_ChunkDesc,
+			ore.m_BlockType, ore.m_BlockMeta,
+			ore.m_MaxHeight, ore.m_NumNests, ore.m_NestSize,
+			seq
+		);
+		seq++;
+	}
+}
+
+
+
+
+
+const cFinishGenOres::OreInfos & cFinishGenOres::DefaultOverworldOres(void)
+{
+	static OreInfos res
+	{
+		// OreType,            OreMeta, MaxHeight, NumNests, NestSize
+		{E_BLOCK_COAL_ORE,     0,       127,       20,       16},
```

Comment:
These are the defaults, the configurability will come in the parent class (not implemented yet) - it will parse a string and replace these defaults with whatever it parses. For the initial version, though, I'll skip :)